permanent_records.gemspec

```diff
@@ -12,27 +14,15 @@ Gem::Specification.new do |s|
     gives you all the scopes you need to work with your data.
   DESCRIPTION
   s.email = 'github@jackcanty.com'
-  s.extra_rdoc_files = [
-    'LICENSE',
-    'README.md'
-  ]
+  s.extra_rdoc_files = %w[LICENSE README.md]
   s.files = `git ls-files -z`.split("\x0").reject do |f|
     f.match(%r{^(test|spec|features)/})
   end
   s.homepage = 'https://github.com/JackDanger/permanent_records'
   s.require_paths = ['lib']
 
-  # For testing against multiple AR versions
-  ver = ENV['AR_TEST_VERSION']
-  ver = ver.dup.chomp if ver
+  s.required_ruby_version = '>= 2.7.8'
 
-  s.add_runtime_dependency 'activerecord',  ver || '>= 5.0.0'
-  s.add_runtime_dependency 'activesupport', ver || '>= 5.0.0'
-  s.add_development_dependency 'database_cleaner', '>= 1.5.1'
-  s.add_development_dependency 'pry-byebug'
-  s.add_development_dependency 'rake' # For Travis-ci
-  s.add_development_dependency 'rspec', '>= 3.5.0'
-  s.add_development_dependency 'rubocop', '~> 0.68.0' # freeze to ensure ruby 2.2 compatibility
-  s.add_development_dependency 'rubocop-performance'
-  s.add_development_dependency 'sqlite3', '~> 1.3.13' # freeze to ensure specs are working
+  s.add_runtime_dependency 'activerecord', ['>= 5.2', '< 7']
+  s.add_runtime_dependency 'activesupport', ['>= 5.2', '< 7']
```

Comment:
What is the reasoning for making this <7?
Maybe it is a config we haven't updated, but we're running the existing version in 7.x for a while now

Reply:
We just hadn't checked it yet so we wanted to be rather safe than sorry ;-) ... If you're running it on production with rails 7 without problems we're happy to add rails 7 and ruby 3.2 to the test matrix and remove the <7 restriction. (/cc @maiwald)

Reply:
Done!